packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -7,7 +7,6 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
 import '../../../service/editor/api_classes.dart';
-import '../../../shared/primitives/utils.dart';
 import 'property_editor_controller.dart';
 
 class PropertyEditorView extends StatelessWidget {
```

Comment:
this widget doesn't provide much value to abstract now. Can we add the `_PropertiesList` directly where `PropertyEditorView` is currently used? That or make `PropertyEditorView` what `_PropertiesList` currently is (effectively deleting `_PropertiesList` and having `PropertyEditorView` contain the body of the current `_PropertiesList`)